insights/parsers/openshift_get.py

```diff
@@ -22,15 +22,15 @@
     True
 """
 
-from .. import YAMLParser, parser
+from .. import YAMLParser, parser, LegacyItemAccess
```

Comment:
@wushiqinlou - better add the `LegacyItemAccess` to `YAMLParser` directly, by doing this, we can avoid forgetting to add the `LegacyItemAccess` to the newly created class(YAMLParser).

And I do suggest you remove the `LegacyItemAcess` from "CobblerSettings" which already added it in this PR.